documentation/practices.md

```diff
@@ -80,5 +80,43 @@ Configuration properties for content URLs are:
 
 For any other CMS, you must update the configurations to reflect the public URLs of the files being retrieved by the particular content provider.
 
+##  Third-Party Dynamic Content Integration Example
+
+**Brand Loyalty Integration for Indi**
+
+The React PWA Reference Storefront is pre-configured with a custom component for Indi integration. This component interacts with various widgets that Indi displays. New widgets may be added in the component as made available. For more information on Indi, see the company [website](https://indi.com/).
+
+The Indi component is a stand-alone component to fetch only the `indi-embed` libraries and display the elements corresponding to the libraries. These elements are passed to the component as a structured list.
+
+This component requires the following parameters:
+* `render`: The list of the elements to display from Indi. For example, `render={['carousel', 'brand', 'product']}`.
```

Comment:
Is there a specific format for this parameter? or should we just say. 
For example, see the `ep.config.json' file settings, `render={['carousel', 'brand', 'product']}`.

Reply:
Yep there's a format. I'll need to keep the example